package.json

```
@@ -18,7 +18,7 @@
     "posttest": "npm run lint",
     "test": "NODE_ENV=test mocha --file './test/helper.js' --recursive \"./src/**/*.test.js\"",
     "test:debug": "NODE_ENV=test mocha --file './test/helper.js' --recursive \"./src/**/*.test.js\" --inspect-brk --exit",
-    "validate": "NODE_ENV=validation mocha --delay --file './test/helper.js' scripts/validation/validate.js",
+    "validate": "NODE_ENV=validation node --unhandled-rejections=strict ./node_modules/mocha/bin/mocha --delay --file './test/helper.js' scripts/validation/validate.js",
```

Using the bin alias is more robust than navigating the filesystem within the module.
```suggestion
    "validate": "NODE_ENV=validation node --unhandled-rejections=strict ./node_modules/.bin/mocha --delay --file './test/helper.js' scripts/validation/validate.js",
```